open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1617,13 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    NextPageToken:
+      description:
+        An opaque next page token, when non-empty this indicates that more results can be returned by server,
+        and when empty this indicates the returned results are complete.
```

Comment:
I think we do not need to do that. We did it for pageToken query parameter for backwards compatibility. But the fact that not returning this value is already backwards compatible, I think we should not have a special case for empty value.

Reply:
Makes sense, will fix the writing on this.